tests/integration/run_integration_test.sh

```
@@ -21,9 +21,38 @@ set -euo pipefail
+            MODE="long"
+            shift
+            ;;
+        -*)
+            echo -e "\033[0;31m[ERROR]\033[0m Unknown option: $1" >&2
+            exit 1
+            ;;
+        *)
+            if [[ -z "${HTTPDIRFS_BIN}" ]]; then
+                HTTPDIRFS_BIN="$1"
+            else
+                echo -e "\033[0;31m[ERROR]\033[0m Multiple binary paths specified or extra arguments: $@" >&2
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)